packages/components/src/navigation/Stack.test.tsx

```diff
@@ -44,37 +44,36 @@ beforeAll(() => {
 describe('stack push and pop tests', () => {
   let texts: string[];
   let stackItems: React.ReactNode[];
-  let stack: ReactWrapper;
+  let rerenderFxn;
```

Comment:
`rerenderFxn` isn't a great name - could just call it `rerender`, type it as `(ui: React.ReactElement) => void` instead of including `Fxn` as a suffix (more commonly I see just `Fn` as well).

I'd just name it `rerender` here.